src/groupByKey.ts

```diff
@@ -2,6 +2,20 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+interface SortItem {
+  [key: string]: Item[];
+}
+
+export function groupByKey(items: Item[], field: keyof Item): SortItem {
+  const sortByKey: Item = { };
```

Comment:
```suggestion
  const sortByKey: SortItem = { };
```
Your function returns the type `SortItem`, but your `sortByKey` variable is of another type. Fix this